ingest/src/main/java/com/connexta/ingest/IngestApplication.java

```diff
@@ -1,18 +1,9 @@
 /*
- * Copyright (c) Connexta
- *
- * This is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Lesser General Public License as
- * published by the Free Software Foundation, either version 3
- * of the License, or any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Lesser General Public License for more details. A copy of the
- * GNU Lesser General Public License is distributed along with this
- * program and can be found at http://www.gnu.org/licenses/lgpl.html.
- */
+Copyright (c) 2019 Connexta, LLC
```

Comment:
Should we format the header like this instead:
```java
/*
 * Copyright (c) 2019 Connexta, LLC
 * Released under the GNU Lesser General Public License version 3; see
 * https://www.gnu.org/licenses/lgpl-3.0.html
 */
```
That's how Apache does it.

Reply:
This was the format that I was given.

Reply:
I'm challenging that. I will ask Eric.

Reply:
Let's block this PR until we have answered this question because this PR isn't blocking any other tasks.

Reply:
@lessarderic said he likes 
```java
/*
 * Copyright (c) 2019 Connexta, LLC
 *
 * Released under the GNU Lesser General Public License version 3; see
 * https://www.gnu.org/licenses/lgpl-3.0.html
 */
``` better. This PR should be updated to do that instead.

Reply:
Great! Thanks for following up on this. I'll update it.